Makefile

```diff
@@ -9,6 +9,10 @@ CODEGEN         := pulumi-gen-${PACK}
 
 WORKING_DIR     := $(shell pwd)
 
+GOPATH 		 ?= "$(HOME)/go"
```

Comment:
Any reason why we are adding GOPATH / GOBIN here?

Reply:
Making the install work the same as our other go-based providers - rather than having to add ~/awsx/dist to the user's path. This logic is taken from the go docs on how to establish the GOBIN
> Executables (main packages) are installed to the directory named by the GOBIN environment variable, which defaults to $GOPATH/bin or $HOME/go/bin if the GOPATH environment variable is not set.

Reply:
🙌🙌🙌🙌